tools/ports/sdl2_image.py

```diff
@@ -46,6 +46,8 @@ def get_lib_name(settings):
     libname += '-' + formats
   if settings.PTHREADS:
     libname += '-mt'
+  if settings.SUPPORT_LONGJMP == 'wasm':
+    libname += '-wasm-sjlj'
```

Comment:
Should we add these to the `variants` list above too?

Reply:
There's a lot of missing variants. The one I'm using is called `libSDL2_image_bmp-gif-jpg-png-wasm-sjlj.a`.